index.ts

```diff
@@ -220,7 +220,7 @@ const bits2int_modN = (bytes: Uint8Array): bigint => { // int2octets can't be us
 const i2o = (num: bigint): Bytes => n2b(num);           // int to octets
 declare const globalThis: Record<string, any> | undefined;    // Typescript global symbol available in
 const cr: { node?: any; web?: any } = {     // browsers only. Ensure no dependence on @types/dom
-  node: typeof require === 'function' && require('crypto'), // node.js require('crypto')
+  node: typeof require === 'function' && require('node:crypto'), // node.js require('crypto')
```

Comment:
I'm surprised TypeScript isn't complaining about referencing `require` here without first defining it.  You should be getting an error `Cannot find name 'require'.`  My guess is that you have `@types/node` installed which is incorrectly allowing you to use node types in this project.  Consider removing that dependency and adding the following to make this code not error:
```ts
declare const require: unknown
```